### PR TITLE
changelog: update change log for 0.7.0 release

### PR DESCRIPTION
**Pull request by fanminshi**
[skip ci]

### CODE REVIEW

**Comment by fanminshi** [comment]
please let me know if I have missed anything.

**Comment by fanminshi** [comment]
cc/ @hongchaodeng @hasbro17

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,48 @@
 ## [Unreleased]
 
+### Decoupling Backup and Restore Logic from Etcd Operator
```
Comment (hongchaodeng) [inline CHANGELOG.md]: Just reference to the issue. But highlight what users need to do here.

**Comment by fanminshi** [comment]
All fixed. PTAL cc/ @hongchaodeng

**Comment by hongchaodeng** [comment]
lgtm

**Pull request merged**